besu/src/test/java/org/hyperledger/besu/cli/operator/OperatorSubCommandTest.java

```diff
@@ -280,34 +294,40 @@ public void shouldGenerateSECP256R1KeysWhenSetAsEcCurve() throws IOException {
 
   @Test
   public void shouldFailIfImportedKeysAreFromDifferentEllipticCurve() {
-    assertThatThrownBy(
-            () ->
-                runCmdAndCheckOutput(
-                    cmd(),
-                    "/operator/config_import_keys_secp256r1_invalid_keys.json",
-                    tmpOutputDirectoryPath,
-                    "genesis.json",
-                    true,
-                    asList("key.pub", "key.priv")))
-        .isInstanceOf(CommandLine.ExecutionException.class)
-        .hasMessageEndingWith(
+    final URL configFilePath =
+        this.getClass().getResource("/operator/config_import_keys_secp256r1_invalid_keys.json");
+    parseCommand(
+        cmd(
+                OperatorSubCommand.COMMAND_NAME,
+                OperatorSubCommand.GENERATE_BLOCKCHAIN_CONFIG_SUBCOMMAND_NAME,
+                "--config-file",
+                configFilePath.getPath(),
+                "--to",
+                tmpOutputDirectoryPath.toString())
+            .args(cmd().argsArray())
+            .argsArray());
+    assertThat(commandErrorOutput.toString(UTF_8).trim())
+        .endsWith(
             "0xb295c4242fb40c6e8ac7b831c916846050f191adc560b8098ba6ad513079571ec1be6e5e1a715857a13a91963097962e048c36c5863014b59e8f67ed3f667680 is not a valid public key for elliptic curve secp256r1");
   }
 
   @Test
   public void shouldFailIfNoConfigSection() {
-    assertThatThrownBy(
-            () ->
-                runCmdAndCheckOutput(
-                    cmd(),
-                    "/operator/config_no_config_section.json",
-                    tmpOutputDirectoryPath,
-                    "genesis.json",
-                    true,
-                    asList("key.pub", "key.priv"),
-                    Optional.of(new SECP256K1())))
-        .isInstanceOf(CommandLine.ExecutionException.class)
-        .hasMessageEndingWith("Missing config section in config file");
+
+    final URL configFilePath =
+        this.getClass().getResource("/operator/config_no_config_section.json");
+    parseCommand(
+        cmd(
+                OperatorSubCommand.COMMAND_NAME,
+                OperatorSubCommand.GENERATE_BLOCKCHAIN_CONFIG_SUBCOMMAND_NAME,
+                "--config-file",
+                configFilePath.getPath(),
+                "--to",
+                tmpOutputDirectoryPath.toString())
+            .args(cmd().argsArray())
+            .argsArray());
+    assertThat(commandErrorOutput.toString(UTF_8).trim())
+        .endsWith("Missing config section in config file");
```

Comment:
It looks like all these updated tests are doing basically the same thing: asserting that a specific error message is sent to the error output. It is probably worth having a helper method that takes the config file path and the expected error msg as parameters so we don't end up with a lot of duplication.
```

Reply:
Done, thanks for the feedback